src/Sdk/DTPipelines/Pipelines/ObjectTemplating/PipelineTemplateEvaluator.cs

```diff
@@ -183,6 +183,33 @@ public class PipelineTemplateEvaluator
             return result;
         }
 
+        public DictionaryContextData EvaluateCompositeOutputs(
```

Comment:
This should be in ActionManifestManager.cs instead. For example see `EvaluateDefaultInput`